tests/test_hyperframe.py

```diff
@@ -0,0 +1,55 @@
+import gzip
+import json
+
+import pandas as pd
+import pytest
+from bs4 import BeautifulSoup
+from pandas._testing import assert_frame_equal
+
+from hyperframe.hyperpattern import HyperPatternHunter
```

Comment:
Could be worth testing `find_and_create_hyperframes` instead of `HyperPatternHunter`, given that the code here basically reimplements that.  Also it'll test if any change to the souping of the html has been made in that function.

Reply:
good idea https://github.com/sam-lawrence/HyperFrame/pull/8/commits/272d30ec62b8ac066ca08c910007ab8c329877c9